### DISCUSSION TITLE
Is there any Sans font?

### DISCUSSION THREAD

**Discussion by azzamsa** [subframe7536/maple-font#301] [Q&A] [answered]
Hi, 👋🏽 

I love maple font so much that I wanted to use the Sans version of this.
Is there any Sans font (non-mono-spaced) that looks similar to maple?

Thanks for maple 🧁

**Comment by subframe7536** [comment, marked as answer]
Thanks for your love ❤️ But saddly, creating a sans-serif version of Maple Mono is beyond the scope of this project.

I had a similar thought before and even merged and modified a raw sans-serif font for Chinese environment. However, after evaluation, I found that the effort and time required to create an ideal version far exceeded what I was willing to commit to.

Therefore, while I can't guarantee I won't make a sans-serif version, I don't have plans to do so in the near future.

**Reply by azzamsa** [reply]
Thanks! 🧁 

I think I will just use Maple mono as is for all other apps.
